src/components/Actions.tsx

```diff
@@ -60,6 +60,10 @@ export const SelectedShapeActions = ({
 
   return (
     <div className="panelColumn">
+      {(elementType === "freedraw" ||
+        targetElements.some((element) => element.type === "freedraw")) &&
```

Comment:
Why limit this feature to freedraw? Why not also add `elementType === "arrow" || elementType === "line"`?

Reply:
Because these elementTypes all have different properties like fill, stroke style, sloppiness, edges or arrowheads which the "freedraw" mode should not care about.

If you want to go that far, you might as well include pens for shapes...